share/mk/bsd.cheri.mk

```diff
@@ -120,9 +120,6 @@ ROOTOBJDIR=	${OBJTOP}/obj-libcheri
 .ifdef CHERI_USE_CAP_TABLE
 CFLAGS+=	-cheri-cap-table-abi=${CHERI_USE_CAP_TABLE}
 .endif
```

Comment:
This should go

Reply:
Hmm, it's not clear to me if bsd.cheri.mk even gets used anymore (or rather, when it still gets used).  I think the mips64c128 MACHINE_ARCH doesn't use it but depends on bsd.cpu.mk only.

See this bit at the top of bsd.cheri.mk:

```
.if ${MACHINE_ARCH:Mmips*} && (!${MACHINE_ARCH:Mmips*c*} || defined(COMPAT_CHERI))
```

So only hybrid MIPS or the libcheri built along with hybrid MIPS uses bsd.cheri.mk I think?  I think the purecap MIPS world no longer uses this at all.  I wonder how far away we are from disabling it for the hybrid MIPS world as well.  Probably a @brooksdavis question.

Reply:
We're at the point where we should make a push to kill bsd.cheri.mk (probably after Thursday though).